cldfbench_doreco.py

```diff
@@ -314,172 +315,174 @@ def create_schema(self, cldf):
+                'datatype': 'string',
             },
             {
                 'name': 'Filename',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'Speaker_ID',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'start',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'end',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'ref',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'tx',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'ft',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'wd_ID',
                 'propertyUrl': 'http://cldf.clld.org/v1.0/terms.rdf#id',
             },
             {
                 'name': 'wd',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'mb_ID',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'mb',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'doreco-mb-algn',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'ps',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'gl',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'ph_id',
-                'datatype': 'str',
+                'datatype': 'string',
             },
             {
                 'name': 'ph',
-                'datatype': 'str',
+                'datatype': 'string',
             },
         )
 
         T = cldf.add_table(
             'metadata.csv',
             {
                 'name': 'ID',
-                'propertyUrl': 'http://cldf.clld.org/v1.0/terms.rdf#id',
             },
             {
                 'name': 'Filename',
-                'propertyUrl': 'http://cldf.clld.org/v1.0/terms.rdf#name',
+                'propertyUrl': 'http://cldf.clld.org/v1.0/terms.rdf#id',
             },
             {
                 'name': 'spk_code',
-                'datatype': 'int',
+                'datatype': 'string',
             },
             {
                 'name': 'spk_age',
-                'datatype': 'int',
+                'datatype': 'string',
```

Comment:
We might want to normalize the content of this column to actually get integers here.

Reply:
Currently, there are some lists of integers, separated by slahes. Is there some way to adapt to this structure? `spk_code` and some other columns are structured identically.

Reply:
We could try
```json
{
  "separator": "/",
  "datatype": "int"
}
```
and then assign `row['spk_age'].split('/')`.